Fluent/Post_Porcess.ipynb

```diff
@@ -8,53 +8,97 @@
     "\n",
     "<img src=\"./Images/ClCdvsAlpha.png\\\" width=\"800\">\n",
-    "Figure 4: Variation of mean Lift and Drag coefficients with $\\alpha$ for flow over rotating cylinder with Re = 100.\n",
-    "\n",
-    "Results in Fig. 4 show that as cylinder rotating rate $\\alpha$ increases, Lift coefficient also increases. The cylinder is rotating counterclockwise. The fluid at the bottom gains velocity and the pressure here reduces compared to pressure at the top of the cylinder. This difference in pressure creates a downward acting lift force on the cylinder. This force increases with increase in $\\alpha$. On the other hand, Drag coefficient decreases with increasing $\\alpha$. Drag force is due to difference in pressure in the streamwise direction. This pressure difference reduces with rotation and mixing. This explains the reduction in $C_D$ with increase in $\\alpha$. \n",
+    "Figure 6: Variation of mean lift and drag coefficients with $\\alpha$ for flow over rotating cylinder with Re = 100.\n",
     "\n",
-    "Furthermore the streamlies around the cylinder are visualized in Fig. 5. Due to counterclockwise rotation of the cylinder, streamlines stack at the bottom of the cylinder. As  $\\alpha$ increases, the stagnation point moves above the centerline of the cylinder. \n",
-    "\n",
-    "<img src=\"./Images/streamlines.png\\\" width = \\\"900\\\">\n",
-    "Fig. 5: Visualization of streamlines around the rotating cylinder with Re = 100.\n",
+    "Results in Fig. 6 show that as cylinder rotating rate $\\alpha$ increases, lift coefficient also increases. The cylinder is rotating counterclockwise. The fluid at the bottom gains velocity and the pressure here reduces compared to pressure at the top of the cylinder. This difference in pressure creates a downward acting lift force on the cylinder. This force increases with increase in $\\alpha$. On the other hand, drag coefficient decreases with increasing $\\alpha$. Drag force is due to difference in pressure in the streamwise direction. This pressure difference reduces with rotation and mixing. This explains the reduction in $C_D$ with increase in $\\alpha$. \n",
     "\n",
     "The vortex shedding has a shedding frequency. A non-dimensional number, Strouhal number (St) is associated with this frequency. Strouhal number is defined as \n",
```

Comment:
Revise the content of this section on Strouhal number according to the changes in the language and discussion that I made in earlier sections.